### PR TITLE
Accepted Accommodation & Travel been updated for both Front-end and b…

### PR DESCRIPTION
**Pull request by mthembbu**
…ack-end.

### CODE REVIEW

File: webapp/src/pages/offer/components/OfferComponent.js [hunk 1]
```diff
@@ -147,6 +164,28 @@ class Offer extends Component {
                   : "Not available"}
               </div>
             </div>
+
+            <div class="row pb-5 ">
+              <div class="col-6 font-weight-bold pr-4"  align="right">
+                <div class="form-check accommodation-container">
```
Comment (jaderabbit) [inline webapp/src/pages/offer/components/OfferComponent.js]: This should only show if they've been awarded accomodation.
Is that the case? 

Same with travel_award

File: api/migrations/versions/539d0567cd6c_.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 """empty message
 
+
```
Comment (jaderabbit) [inline api/migrations/versions/539d0567cd6c_.py]: Blank migrations?

File: api/migrations/versions/472bf293e0a1_.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# revision identifiers, used by Alembic.
+revision = '472bf293e0a1'
+down_revision = ('79c61673a487', '7e8bffa88454')
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    pass
```
Comment (jaderabbit) [inline api/migrations/versions/472bf293e0a1_.py]: Also this one?

Comment (mthembbu) [reply api/migrations/versions/472bf293e0a1_.py]: Resolving. Thanks @jaderabbit

Comment (jaderabbit) [reply api/migrations/versions/472bf293e0a1_.py]: DO NOT RESOLVE NOW! IT'S ALREADY DEPLOYED! Eeek

**Pull request merged**